Cargo.toml

```diff
@@ -82,6 +82,8 @@ parquet2 = {path = "src/parquet2"}
 debug = true
 
 [profile.dev]
+debug = "line-tables-only"
+opt-level = 1
```

Comment:
do we really need opt-level=1 in dev? That seems like it'll greatly slow down the build times

Reply:
Left that artifact in by mistake, I meant to push up the optimization level change for build macros